spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/firestore/GcpFirestoreAutoConfiguration.java

```diff
@@ -94,8 +109,17 @@ public FirestoreMappingContext firestoreMappingContext() {
 
 	@Bean
 	@ConditionalOnMissingBean
-	public FirestoreTemplate firestoreTemplate() {
-		// Placeholder for until auto-config is complete.
-		return new FirestoreTemplate(null, "a/b/c/d/e");
+	public FirestoreTemplate firestoreTemplate() throws IOException {
+		ManagedChannel channel = ManagedChannelBuilder
+				.forAddress("firestore.googleapis.com", 443)
```

Comment:
This should be configurable via `GcpFirestoreProperties.host` just like in Datastore so that one can switch to an emulator.

Reply:
Done. called it "hostPort" since it includes host:port.